firefox-ios/Storage/DefaultSuggestedSites.swift

```diff
@@ -37,83 +37,64 @@ open class DefaultSuggestedSites {
         "default": [
             SuggestedSiteData(
                 url: "https://m.facebook.com/",
-                bgColor: "0x385185",
-                imageUrl: "asset://suggestedsites_facebook",
-                faviconUrl: "asset://defaultFavicon",
+                faviconUrl: "https://static.xx.fbcdn.net/rsrc.php/v3/yi/r/4Kv5U5b1o3f.png",
                 trackingId: 632,
                 title: .DefaultSuggestedFacebook
             ),
             SuggestedSiteData(
                 url: "https://m.youtube.com/",
-                bgColor: "0xcd201f",
-                imageUrl: "asset://suggestedsites_youtube",
-                faviconUrl: "asset://defaultFavicon",
+                faviconUrl: "https://m.youtube.com/static/apple-touch-icon-180x180-precomposed.png",
                 trackingId: 631,
                 title: .DefaultSuggestedYouTube
             ),
             SuggestedSiteData(
                 url: "https://www.amazon.com/",
-                bgColor: "0x000000",
-                imageUrl: "asset://suggestedsites_amazon",
-                faviconUrl: "asset://defaultFavicon",
+                // FIXME Need the correct favicon URL... grabbed this one from the contile services link
```

Comment:
Amazon famously [doesn't have an Apple Touch Icon](https://www.reddit.com/r/MacOS/comments/qhl25n/why_is_amazon_like_this/)...

Is it safe for us to use the contile link as the favicon URL? I got it from https://contile.services.mozilla.com/v1/tiles

Amazon.com only hosts a low quality .ico and an Open Graph image that's the full "Amazon" text (not ideal for favicon sizes).